### PR TITLE
Improve snapshot tests

### PR DESCRIPTION
**Pull request by tomasmarciulionis**
Tried to use dependency injection, but it seemed that it would be overhead for current state of SDK as no dependency injection is currently not used. So, just refactored tests to be more reusable.

### CODE REVIEW

File: android/app/src/androidTest/java/com/trafi/example/ButtonPreviews.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    fun init() {
+        previewTest = PreviewTest(composeTestRule = composeTestRule)
+    }
+
+    @Test
+    fun buttonTest() {
+        previewTest
+            .renderAndTestView {
+                MaasTheme(cornerRadius = MaasCornerRadius(buttonRadius = 0.dp)) {
+                    Button("Unlock now", onClick = {})
```
Comment (justasm) [inline android/app/src/androidTest/java/com/trafi/example/ButtonPreviews.kt:30]: We could call the preview functions directly here, right? e.g. `ButtonStyledCornerRadiusPreview()`

**Review by justasm — APPROVED** [review]
Not clear how the Badge poster screenshot is related to this PR.

**Comment by tomasmarciulionis** [comment]
> Not clear how the Badge poster screenshot is related to this PR. Also, as @absimas has previously mentioned, you can rebase and / or squash commits to clean up the git history and avoid commits like "WIP".

Removed unnecessary file.

File: android/app/src/androidTest/java/com/trafi/example/ButtonPreviews.kt (hunk 1 above)
Comment (tomasmarciulionis) [reply android/app/src/androidTest/java/com/trafi/example/ButtonPreviews.kt:30]: We should be able to call preview functions directly.

**Pull request merged**